README.md

```diff
@@ -43,6 +43,21 @@ The `pages/api` directory is mapped to `/api/*`. Files in this directory are tre
 
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
 
+## Getting Started With Docker
+First, install ["Docker"](https://www.docker.com/) on your local machine. 
+
+After installing docker, cd into the app directory and run the following commands:
+
+```bash 
+docker-compose build # this will build the application and install dependencies
+```
+
+```bash 
+docker-compose up # this will run the application
```

Comment:
Can you also add the `docker-compose down` command to stop the application. I don't think that  removes the image from the local machine.